### PR TITLE
Increment version to 3.2 alpha for release testing

### PR DESCRIPTION
**Pull request by fonnesbeck**
Can merge this once the feature freeze is in effect (i.e. when #2542 is merged).

### CODE REVIEW

**Comment by junpenglao** [comment]
Should we wait for https://github.com/pymc-devs/pymc3/pull/2253?

**Comment by fonnesbeck** [comment]
If its ready, but should not hold up the release, I think.

File: docs/source/conf.py [hunk 1]
```diff
@@ -78,9 +78,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '3.1'
+version = '3.2'
 # The full version, including alpha/beta/rc tags.
-release = '3.1'
+release = '3.2.rc1'
```
Comment (twiecki) [inline docs/source/conf.py]: I think just `3.2rc1`

File: setup.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 AUTHOR_EMAIL = 'chris.fonnesbeck@vanderbilt.edu'
 URL = "http://github.com/pymc-devs/pymc3"
 LICENSE = "Apache License, Version 2.0"
-VERSION = "3.1"
+VERSION = "3.2"
```
Comment (twiecki) [inline setup.py]: shouldn't that be 3.2rc1 here too?

File: docs/source/conf.py (hunk 1 above)
Comment (fonnesbeck) [reply docs/source/conf.py]: Just being consistent with past labels.

File: setup.py (hunk 2 above)
Comment (fonnesbeck) [reply setup.py]: I can. Was distinguishing version vs release.

**Comment by fonnesbeck** [comment]
More Travis shenanigans.

**Pull request merged**